tests/e2e/validatecluster/Vagrantfile

```diff
@@ -1,15 +1,24 @@
 ENV['VAGRANT_NO_PARALLEL'] = 'no'
-NODE_ROLES = (ENV['NODE_ROLES'] ||
+NODE_ROLES = (ENV['E2E_NODE_ROLES'] ||
   ["server-0", "server-1", "server-2", "agent-0" ])
-NODE_BOXES = (ENV['NODE_BOXES'] ||
+NODE_BOXES = (ENV['E2E_NODE_BOXES'] ||
   ['generic/ubuntu2004', 'generic/ubuntu2004', 'generic/ubuntu2004', 'generic/ubuntu2004'])
-GITHUB_BRANCH = (ENV['GITHUB_BRANCH'] || "master")
-RELEASE_VERSION = (ENV['RELEASE_VERSION'] || "")
-NODE_CPUS = (ENV['NODE_CPUS'] || 2).to_i
-NODE_MEMORY = (ENV['NODE_MEMORY'] || 2048).to_i
+GITHUB_BRANCH = (ENV['E2E_GITHUB_BRANCH'] || "master")
+RELEASE_VERSION = (ENV['E2E_RELEASE_VERSION'] || "")
+NODE_CPUS = (ENV['E2E_NODE_CPUS'] || 2).to_i
+NODE_MEMORY = (ENV['E2E_NODE_MEMORY'] || 2048).to_i
+CNI = (ENV['E2E_CNI'] || "canal") # canal, cilium and calico supported
 # Virtualbox >= 6.1.28 require `/etc/vbox/network.conf` for expanded private networks 
```

Comment:
I'm aware this wasn't change within this PR, but, what do you think of adding [VBox documentation URL regarding that configuration](https://www.virtualbox.org/manual/ch06.html#network_hostonly)?

Reply:
@jossemargt-3pillar added